blog/wp-content/themes/mts_sociallyviral/options/demo-importer/demo-files/SV1/theme_options.txt
###a:98:{s:8:"last_tab";s:1:"5";s:8:"mts_logo";s:64:"http://demo.mythemeshop.com/sociallyviral/files/2014/10/logo.png";s:11:"mts_favicon";s:67:"http://demo.mythemeshop.com/sociallyviral/files/2014/10/favicon.png";s:14:"mts_touch_icon";s:0:"";s:14:"mts_metro_icon";s:0:"";s:20:"mts_twitter_username";s:0:"";s:14:"mts_feedburner";s:0:"";s:15:"mts_header_code";s:0:"";s:18:"mts_analytics_code";s:0:"";s:23:"mts_pagenavigation_type";s:1:"2";s:15:"mts_ajax_search";s:1:"1";s:14:"mts_responsive";s:1:"1";s:7:"mts_rtl";s:1:"0";s:17:"mts_shop_products";s:1:"9";s:15:"mts_prefetching";s:1:"0";s:13:"mts_lazy_load";s:1:"0";s:20:"mts_lazy_load_thumbs";s:1:"0";s:21:"mts_lazy_load_content";s:1:"0";s:12:"mts_async_js";s:1:"1";s:21:"mts_remove_ver_params";s:1:"1";s:15:"mts_optimize_wc";s:1:"0";s:16:"mts_color_scheme";s:7:"#33bcf2";s:18:"mts_color_scheme_2";s:7:"#f47555";s:10:"mts_layout";s:8:"cslayout";s:14:"mts_background";a:10:{s:5:"color";s:7:"#f0ede9";s:3:"use";s:7:"pattern";s:13:"image_pattern";s:4:"nobg";s:12:"image_upload";s:0:"";s:6:"repeat";s:6:"repeat";s:10:"attachment";s:6:"scroll";s:8:"position";s:8:"left top";s:4:"size";s:5:"cover";s:8:"gradient";a:3:{s:4:"from";s:7:"#ffffff";s:2:"to";s:7:"#000000";s:9:"direction";s:10:"horizontal";}s:8:"parallax";s:1:"0";}s:14:"mts_custom_css";s:0:"";s:12:"mts_lightbox";s:1:"0";s:19:"mts_header_section2";s:1:"1";s:17:"mts_header_search";s:1:"1";s:14:"mts_sticky_nav";s:1:"0";s:20:"mts_show_primary_nav";s:1:"1";s:22:"mts_show_header_social";s:1:"1";s:17:"mts_header_social";a:4:{i:0;a:6:{s:11:"group_title";s:8:"Facebook";s:10:"group_sort";s:1:"1";s:21:"mts_header_icon_title";s:8:"Facebook";s:15:"mts_header_icon";s:8:"facebook";s:20:"mts_header_icon_link";s:1:"#";s:24:"mts_header_icon_bg_color";s:7:"#375593";}i:1;a:6:{s:11:"group_title";s:7:"Twitter";s:10:"group_sort";s:1:"2";s:21:"mts_header_icon_title";s:7:"Twitter";s:15:"mts_header_icon";s:7:"twitter";s:20:"mts_header_icon_link";s:1:"#";s:24:"mts_header_icon_bg_color";s:7:"#0eb6f6";}i:2;a:6:{s:11:"group_title";s:11:"Google Plus";s:10:"group_sort";s:1:"3";s:21:"mts_header_icon_title";s:11:"Google Plus";s:15:"mts_header_icon";s:11:"google-plus";s:20:"mts_header_icon_link";s:1:"#";s:24:"mts_header_icon_bg_color";s:7:"#dd4b39";}i:3;a:6:{s:11:"group_title";s:7:"YouTube";s:10:"group_sort";s:1:"4";s:21:"mts_header_icon_title";s:7:"YouTube";s:15:"mts_header_icon";s:12:"youtube-play";s:20:"mts_header_icon_link";s:1:"#";s:24:"mts_header_icon_bg_color";s:7:"#e32c26";}}s:14:"mts_top_footer";s:1:"0";s:18:"mts_top_footer_num";s:1:"3";s:14:"mts_footer_nav";s:1:"1";s:14:"mts_copyrights";s:58:"Theme by <a href="http://mythemeshop.com/">MyThemeShop</a>";s:15:"mts_home_layout";s:2:"h1";s:24:"mts_home_layout_big_post";s:1:"1";s:19:"mts_featured_slider";s:1:"0";s:23:"mts_featured_slider_num";s:1:"3";s:23:"mts_featured_categories";a:1:{i:0;a:4:{s:11:"group_title";s:0:"";s:10:"group_sort";s:1:"1";s:21:"mts_featured_category";s:6:"latest";s:30:"mts_featured_category_postsnum";s:1:"3";}}s:27:"mts_home_headline_meta_info";a:2:{s:8:"disabled";a:2:{s:6:"author";s:11:"Author Name";s:7:"comment";s:13:"Comment Count";}s:7:"enabled";a:2:{s:8:"category";s:10:"Categories";s:4:"date";s:4:"Date";}}s:22:"mts_single_post_layout";a:2:{s:7:"enabled";a:3:{s:7:"content";s:12:"Post Content";s:7:"related";s:13:"Related Posts";s:6:"author";s:10:"Author Box";}s:8:"disabled";a:1:{s:4:"tags";s:4:"Tags";}}s:26:"mts_related_posts_taxonomy";s:10:"categories";s:20:"mts_related_postsnum";s:1:"6";s:24:"mts_single_headline_meta";s:1:"1";s:29:"mts_single_headline_meta_info";a:4:{s:6:"author";s:1:"1";s:4:"date";s:1:"1";s:8:"category";s:1:"1";s:7:"comment";s:1:"1";}s:14:"mts_breadcrumb";s:1:"1";s:18:"mts_author_comment";s:1:"1";s:16:"mts_comment_date";s:1:"1";s:26:"mts_social_button_position";s:4:"both";s:27:"mts_social_buttons_on_pages";s:1:"0";s:18:"mts_social_buttons";a:1:{s:7:"enabled";a:7:{s:8:"facebook";s:13:"Facebook Like";s:5:"gplus";s:11:"Google Plus";s:7:"twitter";s:7:"Twitter";s:6:"reddit";s:6:"Reddit";s:9:"pinterest";s:9:"Pinterest";s:7:"stumble";s:11:"StumbleUpon";s:5:"email";s:5:"Email";}}s:17:"mts_header_adcode";s:0:"";s:18:"mts_posttop_adcode";s:138:"<a href="https://mythemeshop.com/themes/sociallyviral/"><img src="http://demo.mythemeshop.com/sociallyviral/files/2015/11/728x90.jpg"></a>";s:23:"mts_posttop_adcode_time";s:1:"0";s:18:"mts_postend_adcode";s:138:"<a href="https://mythemeshop.com/themes/sociallyviral/"><img src="http://demo.mythemeshop.com/sociallyviral/files/2015/11/728x90.jpg"></a>";s:23:"mts_postend_adcode_time";s:1:"0";s:20:"mts_sidebar_for_home";s:0:"";s:20:"mts_sidebar_for_post";s:0:"";s:20:"mts_sidebar_for_page";s:0:"";s:23:"mts_sidebar_for_archive";s:0:"";s:24:"mts_sidebar_for_category";s:0:"";s:19:"mts_sidebar_for_tag";s:0:"";s:20:"mts_sidebar_for_date";s:0:"";s:22:"mts_sidebar_for_author";s:0:"";s:22:"mts_sidebar_for_search";s:0:"";s:24:"mts_sidebar_for_notfound";s:0:"";s:20:"mts_sidebar_for_shop";s:0:"";s:23:"mts_sidebar_for_product";s:0:"";s:32:"mts_navbar_primary-menu_for_home";s:0:"";s:26:"mts_navbar_mobile_for_home";s:0:"";s:32:"mts_navbar_primary-menu_for_post";s:0:"";s:26:"mts_navbar_mobile_for_post";s:0:"";s:32:"mts_navbar_primary-menu_for_page";s:0:"";s:26:"mts_navbar_mobile_for_page";s:0:"";s:35:"mts_navbar_primary-menu_for_archive";s:0:"";s:29:"mts_navbar_mobile_for_archive";s:0:"";s:36:"mts_navbar_primary-menu_for_category";s:0:"";s:30:"mts_navbar_mobile_for_category";s:0:"";s:31:"mts_navbar_primary-menu_for_tag";s:0:"";s:25:"mts_navbar_mobile_for_tag";s:0:"";s:32:"mts_navbar_primary-menu_for_date";s:0:"";s:26:"mts_navbar_mobile_for_date";s:0:"";s:34:"mts_navbar_primary-menu_for_author";s:0:"";s:28:"mts_navbar_mobile_for_author";s:0:"";s:34:"mts_navbar_primary-menu_for_search";s:0:"";s:28:"mts_navbar_mobile_for_search";s:0:"";s:36:"mts_navbar_primary-menu_for_notfound";s:0:"";s:30:"mts_navbar_mobile_for_notfound";s:0:"";s:32:"mts_navbar_primary-menu_for_shop";s:0:"";s:26:"mts_navbar_mobile_for_shop";s:0:"";s:35:"mts_navbar_primary-menu_for_product";s:0:"";s:29:"mts_navbar_mobile_for_product";s:0:"";s:19:"mts_typography_sets";a:1:{i:0;s:5:"latin";}s:29:"google_typography_collections";a:14:{i:0;a:11:{s:12:"preview_text";s:4:"Logo";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"44px";s:10:"font_color";s:7:"#33bcf2";s:13:"css_selectors";s:7:"#logo a";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:4:"Logo";}i:1;a:11:{s:12:"preview_text";s:15:"Navigation Font";s:13:"preview_color";s:4:"dark";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"18px";s:10:"font_color";s:7:"#ffffff";s:13:"css_selectors";s:48:".primary-navigation li, .primary-navigation li a";s:14:"additional_css";s:26:"text-transform: uppercase;";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:15:"Navigation Font";}i:2;a:11:{s:12:"preview_text";s:18:"Home Article Title";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"24px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:40:".latestPost .title, .latestPost .title a";s:14:"additional_css";s:18:"line-height: 36px;";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:18:"Home Article Title";}i:3;a:11:{s:12:"preview_text";s:20:"Single Article Title";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"38px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:13:".single-title";s:14:"additional_css";s:18:"line-height: 53px;";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:20:"Single Article Title";}i:4;a:11:{s:12:"preview_text";s:12:"Content Font";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"16px";s:10:"font_color";s:7:"#707070";s:13:"css_selectors";s:4:"body";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:12:"Content Font";}i:5;a:11:{s:12:"preview_text";s:12:"Widget Title";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:3:"500";s:9:"font_size";s:4:"18px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:21:".widget .widget-title";s:14:"additional_css";s:26:"text-transform: uppercase;";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:12:"Widget Title";}i:6;a:11:{s:12:"preview_text";s:12:"Sidebar Font";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"16px";s:10:"font_color";s:7:"#707070";s:13:"css_selectors";s:17:"#sidebars .widget";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:12:"Sidebar Font";}i:7;a:11:{s:12:"preview_text";s:11:"Footer Font";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"14px";s:10:"font_color";s:7:"#707070";s:13:"css_selectors";s:15:".footer-widgets";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:11:"Footer Font";}i:8;a:11:{s:12:"preview_text";s:10:"Content H1";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"38px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:2:"h1";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:10:"Content H1";}i:9;a:11:{s:12:"preview_text";s:10:"Content H2";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"34px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:2:"h2";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:10:"Content H2";}i:10;a:11:{s:12:"preview_text";s:10:"Content H3";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"30px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:2:"h3";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:10:"Content H3";}i:11;a:11:{s:12:"preview_text";s:10:"Content H4";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"28px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:2:"h4";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:10:"Content H4";}i:12;a:11:{s:12:"preview_text";s:10:"Content H5";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"24px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:2:"h5";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:10:"Content H5";}i:13;a:11:{s:12:"preview_text";s:10:"Content H6";s:13:"preview_color";s:5:"light";s:11:"font_family";s:6:"Roboto";s:12:"font_variant";s:6:"normal";s:9:"font_size";s:4:"20px";s:10:"font_color";s:7:"#555555";s:13:"css_selectors";s:2:"h6";s:14:"additional_css";s:0:"";s:11:"backup_font";s:0:"";s:7:"default";s:4:"true";s:16:"collection_title";s:10:"Content H6";}}s:15:"nhp-opts-backup";s:1:"1";}###

planahorro/chat/setup/settings.php
<?php
	/* (c) OSI Codes Inc. */
	/* http://www.osicodesinc.com */
	/****************************************/
	// STANDARD header for Setup
	if ( !is_file( "../web/config.php" ) ){ HEADER("location: install.php") ; exit ; }
	include_once( "../web/config.php" ) ;
	include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Format.php" ) ;
	include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Error.php" ) ;
	include_once( "$CONF[DOCUMENT_ROOT]/API/".Util_Format_Sanatize($CONF["SQLTYPE"], "ln") ) ;
	include_once( "$CONF[DOCUMENT_ROOT]/API/Util_IP.php" ) ;
	include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Security.php" ) ;
	$ses = Util_Format_Sanatize( Util_Format_GetVar( "ses" ), "ln" ) ;
	if ( !$admininfo = Util_Security_AuthSetup( $dbh, $ses ) ){ ErrorHandler( 608, "Invalid setup session or session has expired.", $PHPLIVE_FULLURL, 0, Array() ) ; }
	// STANDARD header end
	/****************************************/

	include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Vals.php" ) ;
	include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Hash.php" ) ;
	if ( is_file( "$CONF[DOCUMENT_ROOT]/API/Util_Extra_Pre.php" ) ) { include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Upload_.php" ) ; }
	else { include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Upload.php" ) ; }
	include_once( "$CONF[DOCUMENT_ROOT]/API/Util_Upload_File.php" ) ;
	include_once( "$CONF[DOCUMENT_ROOT]/API/Depts/get.php" ) ;

	$https = "" ;
	if ( isset( $_SERVER["HTTP_CF_VISITOR"] ) && preg_match( "/(https)/i", $_SERVER["HTTP_CF_VISITOR"] ) ) { $https = "s" ; }
	else if ( isset( $_SERVER["HTTP_X_FORWARDED_PROTO"] ) && preg_match( "/(https)/i", $_SERVER["HTTP_X_FORWARDED_PROTO"] ) ) { $https = "s" ; }
	else if ( isset( $_SERVER["HTTPS"] ) && preg_match( "/(on)/i", $_SERVER["HTTPS"] ) ) { $https = "s" ; }

	$action = Util_Format_Sanatize( Util_Format_GetVar( "action" ), "ln" ) ;
	$jump = ( Util_Format_Sanatize( Util_Format_GetVar( "jump" ), "ln" ) ) ? Util_Format_Sanatize( Util_Format_GetVar( "jump" ), "ln" ) : "eips" ;
	$deptid = Util_Format_Sanatize( Util_Format_GetVar( "deptid" ), "ln" ) ;

	if ( !isset( $CONF["cookie"] ) ) { $CONF["cookie"] = "on" ; }

	$error = "" ;

	$deptinfo = Depts_get_DeptInfo( $dbh, $deptid ) ;
	LIST( $your_ip, $null ) = Util_IP_GetIP( "" ) ;

	$cookie_off = ( $CONF["cookie"] == "off" ) ? "checked" : "" ;
	$cookie_on = ( $cookie_off == "checked" ) ? "" : "checked" ;

	// auto write conf file if variables do not exist
	if ( !isset( $CONF["API_KEY"] ) )
	{
		$CONF["API_KEY"] = Util_Format_RandomString( 10 ) ;
		$error = ( Util_Vals_WriteToConfFile( "API_KEY", $CONF["API_KEY"] ) ) ? "" : "Could not write to config file." ;
	}

	$departments = Depts_get_AllDepts( $dbh ) ;
	$timezones = Util_Hash_Timezones() ;
	$vars = Util_Format_Get_Vars( $dbh ) ;
	$charset = ( isset( $vars["char_set"] ) && $vars["char_set"] ) ? unserialize( $vars["char_set"] ) : Array(0=>"UTF-8") ;
?>
<?php include_once( "../inc_doctype.php" ) ?>
<head>
<title> PHP Live! Support <?php echo $VERSION ?> </title>

<meta name="description" content="PHP Live! Support <?php echo $VERSION ?>">
<meta name="keywords" content="powered by: PHP Live!  www.phplivesupport.com">
<meta name="robots" content="all,index,follow">
<meta http-equiv="content-type" content="text/html; CHARSET=utf-8">
<?php include_once( "../inc_meta_dev.php" ) ; ?>

<link rel="Stylesheet" href="../css/setup.css?<?php echo $VERSION ?>">
<script type="text/javascript" src="../js/global.js?<?php echo $VERSION ?>"></script>
<script type="text/javascript" src="../js/setup.js?<?php echo $VERSION ?>"></script>
<script type="text/javascript" src="../js/framework.js?<?php echo $VERSION ?>"></script>
<script type="text/javascript" src="../js/framework_cnt.js?<?php echo $VERSION ?>"></script>
<script type="text/javascript" src="../js/jquery_md5.js?<?php echo $VERSION ?>"></script>
<script type="text/javascript" src="../js/js_cookie.js?<?php echo $VERSION ?>"></script>

<script type="text/javascript">
<!--
	var global_cookie = "<?php echo $CONF["cookie"] ?>" ;

	$(document).ready(function()
	{
		$("body").css({'background': '#8DB26C'}) ;

		init_menu() ;
		toggle_menu_setup( "settings" ) ;

		fetch_eips() ;
		fetch_sips() ;
		show_div( "<?php echo $jump ?>" ) ;

		<?php if ( $action && !$error ): ?>do_alert( 1, "Success" ) ;<?php endif ; ?>
		<?php if ( $action && $error ): ?>do_alert_div( "..", 0, "<?php echo $error ?>" ) ;<?php endif ; ?>
	});

	function fetch_eips()
	{
		$.ajax({
			type: "POST",
			url: "../ajax/setup_actions.php",
			data: "ses=<?php echo $ses ?>&action=eips&"+unixtime(),
			success: function(data){
				print_eips( data ) ;
			}
		});
	}

	function fetch_sips()
	{
		$.ajax({
			type: "POST",
			url: "../ajax/setup_actions.php",
			data: "ses=<?php echo $ses ?>&action=sips&"+unixtime(),
			success: function(data){
				print_sips( data ) ;
			}
		});
	}

	function print_eips( thedata )
	{
		eval( thedata ) ;
		if ( json_data.ips != undefined )
		{
			var ip_string = "<table cellspacing=0 cellpadding=0 border=0 width=\"100%\">" ;
			for ( c = 0; c < json_data.ips.length; ++c )
			{
				var ip = json_data.ips[c]["ip"] ;
				var ip_ = ip.replace( /\./g, "" ) ;

				ip_string += "<tr><td class=\"td_dept_td\" width=\"14\"><div id=\"eip_"+ip_+"\"><a href=\"JavaScript:void(0)\" onClick=\"remove_eip( '"+ip+"' )\"><img src=\"../pics/icons/delete.png\" width=\"14\" height=\"14\" border=\"0\" alt=\"\"></a></div></td><td class=\"td_dept_td\">"+ip+"</td></tr>" ;
			}
			if ( !c )
				ip_string += "<tr><td class=\"td_dept_td\">Blank results.</td></tr>" ;
		}
		ip_string += "</table>" ;
		$('#eips').html( ip_string ) ;
	}

	function print_sips( thedata )
	{
		eval( thedata ) ;
		if ( json_data.ips != undefined )
		{
			var ip_string = "<table cellspacing=0 cellpadding=0 border=0 width=\"100%\">" ;
			for ( c = 0; c < json_data.ips.length; ++c )
			{
				var ip = json_data.ips[c]["ip"] ;
				var ip_ = ip.replace( /\./g, "" ) ;

				ip_string += "<tr><td class=\"td_dept_td\" width=\"14\"><div id=\"sip_"+ip_+"\"><a href=\"JavaScript:void(0)\" onClick=\"remove_sip( '"+ip+"' )\"><img src=\"../pics/icons/delete.png\" width=\"14\" height=\"14\" border=\"0\" alt=\"\"></a></div></td><td class=\"td_dept_td\">"+ip+"</td></tr>" ;
			}
			if ( !c )
				ip_string += "<tr><td class=\"td_dept_td\">Blank results.</td></tr>" ;
		}
		ip_string += "</table>" ;
		$('#sips').html( ip_string ) ;
	}

	function add_eip()
	{
		var ip = $('#ip_exclude').val().replace( /[^0-9.]/g, "" ) ;
		$('#ip_exclude').val( ip ) ;

		if ( !ip )
			do_alert( 0, "Blank IP field is invalid." ) ;
		else
		{
			$.ajax({
				type: "POST",
				url: "../ajax/setup_actions.php",
				data: "ses=<?php echo $ses ?>&action=add_eip&ip="+ip+"&"+unixtime(),
				success: function(data){
					eval(data) ;
					if ( json_data.status )
						fetch_eips() ;
					else
						do_alert( 0, "IP ("+ip+") already excluded." ) ;

					$('#ip_exclude').val('') ;
				}
			});
		}
	}

	function remove_eip( theip )
	{
		var theip_ = theip.replace( /\./g, "" ) ;
		$('#eip_'+theip_).html( "<img src=\"../pics/loading_ci.gif\" width=\"16\" height=\"16\" border=\"0\" alt=\"\">" ) ;

		$.ajax({
			type: "POST",
			url: "../ajax/setup_actions.php",
			data: "ses=<?php echo $ses ?>&action=remove_eip&ip="+theip+"&"+unixtime(),
			success: function(data){
				print_eips( data ) ;
			}
		});
	}

	function add_sip()
	{
		var ip = $('#ip_spam').val().replace( /[^0-9.]/g, "" ) ;
		$('#ip_spam').val( ip ) ;

		if ( !ip )
			do_alert( 0, "Blank IP field is invalid." ) ;
		else
		{
			$.ajax({
				type: "POST",
				url: "../ajax/setup_actions.php",
				data: "ses=<?php echo $ses ?>&action=add_sip&ip="+ip+"&"+unixtime(),
				success: function(data){
					eval(data) ;
					if ( json_data.status )
						fetch_sips() ;
					else
						do_alert( 0, "IP ("+ip+") already reported as spam." ) ;

					$('#ip_spam').val('') ;
				}
			});
		}
	}

	function remove_sip( theip )
	{
		var theip_ = theip.replace( /\./g, "" ) ;
		$('#sip_'+theip_).html( "<img src=\"../pics/loading_ci.gif\" width=\"16\" height=\"16\" border=\"0\" alt=\"\">" ) ;

		$.ajax({
			type: "POST",
			url: "../ajax/setup_actions.php",
			data: "ses=<?php echo $ses ?>&action=remove_sip&ip="+theip+"&"+unixtime(),
			success: function(data){
				print_sips( data ) ;
			}
		});
	}

	function show_div( thediv )
	{
		var divs = Array( "eips", "sips", "cookie", "profile" ) ;
		for ( c = 0; c < divs.length; ++c )
		{
			$('#settings_'+divs[c]).hide() ;
			$('#menu_'+divs[c]).removeClass('op_submenu_focus').addClass('op_submenu') ;
		}

		$('input#jump').val( thediv ) ;
		$('#settings_'+thediv).show() ;
		$('#menu_'+thediv).removeClass('op_submenu').addClass('op_submenu_focus') ;
	}

	function switch_dept( theobject )
	{
		location.href = "settings.php?ses=<?php echo $ses ?>&deptid="+theobject.value ;
	}

	function update_profile()
	{
		execute = 1 ;
		var inputs = Array( "email", "login" ) ;

		if ( !check_email( $('#email').val() ) ){ do_alert( 0, "Email format is invalid. (example: you@domain.com)" ) ; execute = 0 ; }

		if ( $('#npassword').val() || $('#vpassword').val() )
		{
			if ( $('#npassword').val() != $('#vpassword').val() ){ do_alert( 0, "New Password and Verify Password does not match." ) ; execute = 0 ; }
		}

		if ( execute ){ update_profile_doit() ; } ;
	}

	function update_profile_doit()
	{
		var json_data = new Object ;
		var unique = unixtime() ;

		var email = $('#email').val() ;
		var login = $('#login').val() ;
		var npassword = phplive_md5( $('#npassword').val() ) ;
		var vpassword = phplive_md5( $('#vpassword').val() ) ;
		var md5_password = phplive_md5( npassword+vpassword+Cookies.get("phplive_token") ) ;

		$.ajax({
		type: "POST",
		url: "../ajax/setup_actions.php",
		data: "action=update_profile&ses=<?php echo $ses ?>&email="+email+"&login="+login+"&npassword="+npassword+"&vpassword="+vpassword+"&md5_password="+md5_password+"&"+unique,
		success: function(data){
			eval( data ) ;
			if ( json_data.status )
			{
				$('#npassword').val('') ;
				$('#vpassword').val('') ;
				do_alert( 1, "Success" ) ;
			}
			else
				do_alert( 0, json_data.error ) ;

		},
		error:function (xhr, ajaxOptions, thrownError){
			do_alert( 0, "Connection to server was lost.  Please reload the page." ) ;
		} });
	}

	function confirm_change( theflag )
	{
		if ( global_cookie != theflag )
		{
			$.ajax({
				type: "POST",
				url: "../ajax/setup_actions.php",
				data: "ses=<?php echo $ses ?>&action=update_cookie&value="+theflag+"&"+unixtime(),
				success: function(data){
					global_cookie = theflag ;
					do_alert( 1, "Success!" ) ;
				}
			});
		}
	}

//-->
</script>
</head>
<?php include_once( "./inc_header.php" ) ?>

		<div class="op_submenu_wrapper">
			<div class="op_submenu" onClick="show_div('eips')" id="menu_eips">Excluded IPs</div>
			<div class="op_submenu" onClick="show_div('sips')" id="menu_sips">Blocked IPs</div>
			<div class="op_submenu" onClick="show_div('cookie')" id="menu_cookie">Cookies</div>
			<?php if ( $admininfo["adminID"] == 1 ): ?><div class="op_submenu" onClick="show_div('profile')" id="menu_profile"><img src="../pics/icons/key.png" width="12" height="12" border="0" alt=""> Setup Profile</div><?php endif ; ?>
			<div class="op_submenu" onClick="location.href='db.php?ses=<?php echo $ses ?>'" id="menu_system">System</div>
			<div style="clear: both"></div>
		</div>

		<form method="POST" action="settings.php?submit" enctype="multipart/form-data">
		<input type="hidden" name="action" value="update">
		<input type="hidden" name="jump" id="jump" value="">
		<input type="hidden" name="ses" value="<?php echo $ses ?>">

		<div style="display: none; margin-top: 25px; text-align: justify;" id="settings_eips">
			To avoid misleading page views when developing a site, exclude internal or company IP from being counted towards the overall footprint report.  Excluded IPs will not be visible on the traffic monitor. URL footprints of Excluded IPs will not be stored in the database and will not count towars the overall <a href="reports_traffic.php?ses=<?php echo $ses ?>">footprint report</a>.

			<div style="margin-top: 25px;">
				<table cellspacing=0 cellpadding=0 border=0 width="100%">
				<tr>
					<td valign="top" nowrap style="padding-right: 25px;">
						<div class="info_info">
							<div>Your IP: <big><b><?php echo $your_ip ?></b></big></div>
							<div style="margin-top: 15px;"><input type="text" name="ip_exclude" id="ip_exclude" size="20" maxlength="45" onKeyPress="return numbersonly(event)"></div>
							<div style="margin-top: 25px;"><input type="button" onClick="add_eip()" value="Add Exclude IP" class="btn"></div>
						</div>
					</td>
					<td valign="top" width="100%">
						<div><div class="td_dept_header">Current Excluded IPs:</div></div>
						<div id="eips" style="max-height: 300px; overflow: auto;"></div>
					</td>
				</tr>
				</table>
			</div>
		</div>

		<div style="display: none; margin-top: 25px;" id="settings_sips">
			Blocked IPs will always see an OFFLINE status icon.  Operators can specify an IP to block during a chat session or you can provide an IP address here.  Blocked IPs will still display on the traffic monitor.

			<div style="margin-top: 25px;">
				<table cellspacing=0 cellpadding=0 border=0 width="100%">
				<tr>
					<td valign="top" nowrap style="padding-right: 25px;">
						<div class="info_info">
							<div>Example: <big><b>123.456.789.101</b></big></div>
							<div style="margin-top: 15px;"><input type="text" name="ip_spam" id="ip_spam" size="20" maxlength="45" onKeyPress="return numbersonly(event)"></div>
							<div style="margin-top: 25px;"><input type="button" onClick="add_sip()" value="Add IP to Block" class="btn"></div>
						</div>
					</td>
					<td valign="top" width="100%">
						<div><div class="td_dept_header">Current Blocked IPs:</div></div>
						<div id="sips" style="max-height: 300px; overflow: auto;"></div>
					</td>
				</tr>
				</table>
			</div>
		</div>

		<div style="display: none; margin-top: 25px;" id="settings_cookie">
			Switch on/off the use of cookies on the visitor chat window.  The cookies provide convenience for the visitor and does not affect the actual chat functions.
			<div style="margin-top: 25px;" class="info_info">
				Cookies set by the system on the visitor chat request window:
				<li style="margin-top: 5px;"> <code>phplive_vname</code> - The visitor's name
				<li> <code>phplive_vemail</code> - The visitor's email address
			</div>

			<div style="margin-top: 25px;">
				<div class="li_op round"><input type="radio" name="cookie" id="cookie_on" value="on" onClick="confirm_change('on')" <?php echo $cookie_on ?>> Set cookies</div>
				<div class="li_op round"><input type="radio" name="cookie" id="cookie_off" value="off" onClick="confirm_change('off')" <?php echo $cookie_off ?>> Do not set cookies</div>
				<div style="clear: both;"></div>
			</div>
		</div>

		<div style="display: none; margin-top: 25px;" id="settings_profile">
			Update the setup admin contact email address and the password to this setup area.  The admin email is used for various areas of the system, such as notification when an error is produced or when updating various setup settings.

			<div style="margin-top: 25px;">
				<input type="hidden" name="login" id="login" value="<?php echo $admininfo["login"] ?>">
				<table cellspacing=0 cellpadding=0 border=0>
				<tr>
					<td class="td_dept_td" width="120">Setup Admin Email</td>
					<td class="td_dept_td"><input type="text" class="input" size="35" maxlength="50" name="email" id="email" value="<?php echo $admininfo["email"] ?>" onKeyPress="return justemails(event)" value=""></td>
				</tr>
				<tr>
					<td colspan="4" style="padding-top: 15px;">
						<div style="font-size: 14px; font-weight: bold;">Update Password (optional)</div>
						<div style="margin-top: 15px;">
							<table cellspacing=0 cellpadding=4 border=0>
							<tr> 
								<td class="td_dept_td" width="120">New Password</td> 
								<td class="td_dept_td"><input type="password" class="input" size="35" maxlength="50" id="npassword"></td> 
							</tr>
							<tr>
								<td class="td_dept_td" width="120">Verify Password</td> 
								<td class="td_dept_td"><input type="password" class="input" size="35" maxlength="50" id="vpassword"></td> 
							</tr>
							</table>
						</div>
					</td>
				</tr>
				<tr> 
					<td></td> 
					<td class="td_dept_td"><input type="button" value="Update Profile" id="btn_submit" onClick="update_profile()" class="btn"></td> 
				</tr> 
				</table>
			</div>
		</div>

		</form>

<?php include_once( "./inc_footer.php" ) ?>

